apps/analytics/urls.py

```diff
@@ -24,7 +24,7 @@
         name="get_challenge_phase_submission_analysis",
     ),
     url(
-        r"^challenge/(?P<challenge_pk>[0-9]+)/challenge_phase/(?P<challenge_phase_pk>[0-9]+)/count$",
+        r"^challenge/(?P<challenge_pk>[0-9]+)/challenge_phase/(?P<version>(v1|v2|v3))/(?P<challenge_phase_pk>[-a-zA-Z0-9_]+)/count$",
```

Comment:
@Kajol-Kumari Why there are 3 versions?

Reply:
2 will be enough to serve our purpose.will remove the third one.